model/push_notification.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 const (
-	PUSH_NOTIFY_APPLE   = "apple"
-	PUSH_NOTIFY_ANDROID = "android"
+	PUSH_NOTIFY_APPLE_V1   = "apple"
+	PUSH_NOTIFY_ANDROID_V1 = "android"
+	PUSH_NOTIFY_APPLE_V2   = "apple2"
+	PUSH_NOTIFY_ANDROID_V2 = "android2"
```

Comment:
Why having it differently? and if that's the case I don't think having a `2` makes it easy to identify